src/utils.ts

```diff
@@ -35,16 +35,7 @@ function bigIntStringify<T>(_key: unknown, value: T) {
  * @param mints
  * @returns
  */
-function getEncodedToken(proofs: Array<Proof>, mint: string, memo?: string): string {
-	const token: Token = {
-		token: [{ mint, proofs }]
-	};
-
-	//add memo if exist
-	if (memo) {
-		token.memo = memo;
-	}
-
+function getEncodedToken(token: Token): string {
```

Comment:
damn, i guess this will break clients again

Reply:
but we have to do it. I should have built it better in the last PR. that is my mistake